week-1/interpret/to-pounds.js

```diff
@@ -1,21 +1,23 @@
-const penceString = "399p";
+const penceString = "399p"; //this defines the value of the pence in a string 
 
 const penceStringWithoutTrailingP = penceString.substring(
   0,
   penceString.length - 1
-);
+); // substring means it extracts a character, it is usually written like this substring(indexStart) or substring(indexStart,indexEnd)
+// in this example it is substring (0, penceString.lenght -1) which means 0 is the first character which is 3 in this example and -1 means the last character is being cut off so it will be 399
 
-const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0");
+const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0"); // this means it will add a character or value to the string until it reaches the length required
 const pounds = paddedPenceNumberString.substring(
   0,
   paddedPenceNumberString.length - 2
-);
+); 
 
 const pence = paddedPenceNumberString
   .substring(paddedPenceNumberString.length - 2)
-  .padEnd(2, "0");
+  .padEnd(2, "0"); // substring in this example means it will cut off the first 2 characters fromt he start so it will be 3
+  // padend means it will add the specified character starting from the end until it reaches the desired number
 
-console.log(`£${pounds}.${pence}`);
+console.log(`£${pounds}.${pence}`); // this will log the string value of pound and pence 
```

Comment:
You've done well in explaining most of the lines. This answer could be improved by explaining how the variables will have changed at the end of each expression.

Reply:
added some more comment on my code explaining the values